docs/3.api/1.components/4.nuxt-link.md

```diff
@@ -37,26 +42,46 @@ In this example, we pass the `id` param to link to the route `~/pages/posts/[id]
 </template>
 ```
 
+```html [(Renders as) index.html]
+<a href="/posts/123">Post 123</a>
+```
+::
+
 ::tip
 Check out the Pages panel in Nuxt DevTools to see the route name and the params it might take.
 ::
 
-### Handling 404s
+### Handling Static File and Cross-App Links
 
-When using `<NuxtLink>` for `/public` directory files or when pointing to a different app on the same domain, you should use the `external` prop.
+When linking to static files in the `/public` directory or to another application hosted on the same domain, it’s essential to bypass Vue Router's internal routing mechanism to avoid unexpected 404 errors. In such cases, you can use the `external` prop with `<NuxtLink>`.
```

Comment:
```suggestion
By default, `<NuxtLink>` uses Vue Router's client side navigation for relative route. When linking to static files in the `/public` directory or to another application hosted on the same domain, it might result in unexpected 404 errors because they are not part of the client routes. In such cases, you can use the `external` prop with `<NuxtLink>` to bypass Vue Router's internal routing mechanism.
```

Would this explanation more clear? (or too technical?)